doc/dst-host-192.68.1.1-and-greater-100.md

```diff
@@ -54,18 +54,28 @@ end
 ```
 local cast = require("ffi").cast
 return function(P,length)
-   if length < 100 then return false end
+   if length < 34 then return false end
```

Comment:
It seems like we are missing out on an optimization here.  I wonder why that is?